moderngl_window/atlas/base.py

```diff
@@ -44,21 +45,21 @@ class AtlasImage(BaseImage):
     """An atlas image using Pillow"""
 
     def __init__(self, image: Image):
-        self.image = image
+        self._image = image
 
     @property
     def width(self) -> int:
-        return self._image.size[0]
+        return self._image.width
```

Comment:
PIL.Image.Image has `width` and `height` proterty, so it is a mistakes to not use them

Reply:
It's probably better for readability.